connectors/spark/src/test/java/io/unitycatalog/connectors/spark/SparkIntegrationTest.java

```diff
@@ -110,65 +98,53 @@ public void testDeltaPathTable() throws IOException, ApiException {
     }
 
     @Test
-    public void testManagedParquetTable() throws ApiException, IOException, ParseException {
+    public void testCredentialParquet() throws ApiException, IOException {
```

Comment:
why not just `testParquet()`. it is doing the full read/write test, not just credentials... isnt it?

Reply:
There is already `testParquetReadWrite` test in this file. The difference of this new test is it uses s3 so that it tests credentials.